src/components/helpers/BoxLoader/BoxLoader.scss

```diff
@@ -30,7 +30,7 @@
 
   &.theme--dark {
     .mask {
-      background-color: var(--v-torusBlack-base);
```

Comment:
Please use spinnerBackground on changing the background color. many pages using this is using torusBlack